src/Controls/src/Core/BindableProperty.cs

```diff
@@ -41,6 +41,7 @@ public sealed class BindableProperty
 		static readonly Dictionary<Type, TypeConverter> KnownTypeConverters = new Dictionary<Type, TypeConverter>
 		{
 			{ typeof(Uri), new UriTypeConverter() },
+			{ typeof(Maui.Graphics.Color), new ColorTypeConverter() },
```

Comment:
This is the main change.

Reply:
That list is only for types we can’t attribute. Don’t we have a converter attribute on colour?

Reply:
Talked about this with Stephane and is ok to merge, Shane could you review it again?